src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -1079,6 +1079,7 @@ Collection<CompactionAggregate.UnifiedAggregate> getCompactionAggregates(Arena a
                 logger.trace("Creating compaction aggregate with sstable set {}", sstables);
 
 
+            // overlap set with larger sstables are priotized. note that different overlap set may include deduplicated sstable
```

Comment:
The first part of the comment is no longer true. The same sstable can be included in different overlap sets, but only if they are adjacent.